modules/quality/include/opencv2/quality/QualityBase.hpp

```diff
@@ -0,0 +1,68 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_QUALITYBASE_HPP
+#define OPENCV_QUALITYBASE_HPP
+
+#include <vector>
+#include <opencv2/core.hpp>
+
+/**
+@defgroup quality Image Quality Analysis (IQA) API
+*/
+
+namespace cv {
+namespace quality {
+namespace detail {
+    /** @brief Type of output quality maps (if generated by the algorithm) */
+    using quality_map_type = UMat;
+}   // detail
+
+//! @addtogroup quality
+//! @{
+
+	/************************************ Quality Base Class ************************************/
+
+	class CV_EXPORTS_W QualityBase
+        : public virtual Algorithm
+	{
+	public:
+
+        /**
+		@brief Destructor
```

Comment:
Please fix indentation:
- no additional spaces in **namespaces**
- **no tabs** (anywhere)